gst/nnstreamer/nnstreamer_plugin_api_filter.h

```diff
@@ -28,23 +28,17 @@
 
 /** Macros for accelerator types */
 #define ACCL_NONE_STR "none"
+#define ACCL_DEFAULT_STR  "default"
 #define ACCL_AUTO_STR "auto"
 #define ACCL_CPU_STR  "cpu"
+#define ACCL_CPU_NEON_STR  "cpu.neon"
 #define ACCL_GPU_STR  "gpu"
 #define ACCL_NPU_STR  "npu"
-#define ACCL_NEON_STR "neon"
-#define ACCL_SRCN_STR "srcn"
-#define ACCL_DEF_STR  "default"
-
-#define REGEX_ACCL_EACH_ELEM(ELEM) "[^!]" ELEM "[,]?"
-
-#define REGEX_ACCL_AUTO REGEX_ACCL_EACH_ELEM(ACCL_AUTO_STR)
-#define REGEX_ACCL_CPU  REGEX_ACCL_EACH_ELEM(ACCL_CPU_STR)
-#define REGEX_ACCL_GPU  REGEX_ACCL_EACH_ELEM(ACCL_GPU_STR)
-#define REGEX_ACCL_NPU  REGEX_ACCL_EACH_ELEM(ACCL_NPU_STR)
-#define REGEX_ACCL_NEON REGEX_ACCL_EACH_ELEM(ACCL_NEON_STR)
-#define REGEX_ACCL_SRCN REGEX_ACCL_EACH_ELEM(ACCL_SRCN_STR)
-#define REGEX_ACCL_DEF  REGEX_ACCL_EACH_ELEM(ACCL_DEF_STR)
+#define ACCL_NPU_MOVIDIUS_STR  "npu.movidius"
+#define ACCL_NPU_EDGE_TPU_STR  "npu.edgetpu"
+#define ACCL_NPU_VIVANTE_STR  "npu.vivante"
+#define ACCL_NPU_SRCN_STR  "npu.srcn" /** srcn hardware supported by nnfw */
```

Comment:
What does `SRCN` stand for?

Reply:
SRCN is one of the backends supported by NNFW-runtime.
check https://review.tizen.org/gerrit/gitweb?p=platform/core/ml/nnfw.git;a=summary for more info.

Reply:
Thx